packages/aws-lambda-otel-extension/opt/otel-extension/external/helper.js

```diff
@@ -10,6 +11,34 @@ const extensionVersion = (() => {
   return require('../version');
 })();
 
+const isJson = (data) => {
+  try {
+    const res = JSON.parse(data);
+    return res;
+  } catch (error) {
+    return null;
+  }
+};
+const stripResponseBlobData = (data) => {
+  const raw = data.responseData;
+
+  if (!isObject(raw)) {
+    delete data.responseData;
+    data.isBodyExcluded = true;
```

Comment:
In this case we do not return `responseData` at all, so I think setting `isBodyExcluded` is not needed (?)